core/src/test/java/org/apache/accumulo/core/clientImpl/ClientContextTest.java

```diff
@@ -170,4 +171,58 @@ public void testGetBatchWriterConfigNotUsingDefaults() {
     assertEquals(expectedDurability, batchWriterConfig.getDurability());
   }
 
+  @Test
+  public void testGetConditionalWriterConfigUsingDefaults() {
+    Properties props = new Properties();
+    ConditionalWriterConfig conditionalWriterConfig =
+        ClientContext.getConditionalWriterConfig(props);
+    assertNotNull(conditionalWriterConfig);
+
+    // If the value of BATCH_WRITER_TIMEOUT_MAX is set to zero, Long.MAX_VALUE is returned.
```

Comment:
```suggestion
    // If the value of CONDITIONAL_WRITER_TIMEOUT_MAX is set to zero, Long.MAX_VALUE is returned.
```

Reply:
There is a second occurrence in this comment that needs to be changed as well but there isn't a good way to suggest a multi-line change here because the extra length pushes words onto the next line.